src/bokeh/settings.py

```diff
@@ -567,6 +567,13 @@ def dev(self) -> bool:
     See the :class:`~bokeh.resources.Resources` class reference for full details.
     """)
 
+    chromedriver_path: PrioritizedSetting[str | None] = PrioritizedSetting("chromedriver_path", "BOKEH_CHROMEDRIVER_PATH", default=None, help="""
+    The name of or full path to chromedriver's executable.
+
+    This is used to allow ``bokeh.io.export`` to work on systems that use a different name
+    for ``chromedriver``, like ``chromedriver-binary`` or ``chromium.chromedriver`` (Snap).
```

Comment:
If I didn't know what "Snap" was already, this parenthetical would be confusing at best. Suggest either expanding it with more context, or removing it.